.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Python CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches:
+    - '**'
+
+jobs:
+  run_tests:
+    name: Tests
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ['3.8']
+        node-version: ['12']
+        toxenv: [js, quality, django22, django30, django31, django32]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Stup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Setup Node
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install Dependencies
+      run: |
+        sudo apt-get update && sudo apt-get install -y gettext
```

Comment:
Should `gettext` be getting update to the latest every time? I am concerning about local environment on different platforms.